src/new/wallet.js

```diff
@@ -737,7 +737,8 @@ class HathorWallet extends EventEmitter {
       chooseInputs = false;
     }
 
-    // Warning: prepareSendTokensData(...) might modify `partialData`.
+    // Warning: prepareSendTokensData(...) might modify `partialData`. It might add inputs in the inputs array
```

Comment:
I agree with you. This is an old method used in many places right now and that will be deprecated soon. I've added a more explicit comment on it but I'd rather let the deprecation happen instead of changing it right now and update all the places using it (there are many places).